src/views/map-view.ts

```diff
@@ -321,6 +320,39 @@ export class MapView<N extends INodeBase, E extends IEdgeBase> implements IOrbVi
           globalPoint: containerPoint,
         });
 
+        if (response.isStateChanged || response.changedSubject) {
+          this._renderer.render(this._graph);
+        }
+      } else if (event.type === 'dblclick' && this._strategy.onMouseDoubleClick) {
```

Comment:
There is a problem with the map view because double click has two actions: One does the event of DOUBLE_CLICK, but the other one does the leaflet zoom (because of the double click).

If there is a strategy and the subject is either `node` or an `edge`, it would be great to prevent the default leaflet zoom operation on double click. If there is a double click on canvas (no subject), the leaflet should do the default action on double click which is a zoom.
You can play with this in the "Map view" example.